datadog/data_source_datadog_cloud_workload_security_agent_rules.go

```diff
@@ -0,0 +1,76 @@
+package datadog
+
+import (
+	"context"
+	"strings"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceDatadogCloudWorkloadSecurityAgentRules() *schema.Resource {
+	return &schema.Resource{
+		Description: "Use this data source to retrieve information about existing cloud workload security agent rules for use in other resources.",
+		ReadContext: dataSourceDatadogCloudWorkloadSecurityAgentRulesRead,
+
+		Schema: map[string]*schema.Schema{
+			// Computed
+			"agent_rules_ids": {
```

Comment:
Whats the use case for having this in a separate list? I think the id's should be stored within each of the `agent_rules`.

Reply:
Yep you're right, just put it inside each agent_rule